ports/wtl/portfile.cmake

```
@@ -1,10 +1,10 @@
 vcpkg_from_sourceforge(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO wtl/WTL%2010
-    REF WTL%2010.0.9163
-    FILENAME "WTL10_9163.zip"
+    REF WTL%2010.0.10077
+    FILENAME "WTL10_1077.zip"
```

```suggestion
    FILENAME "WTL10_1077..zip"
```
This file should be `WTL10_1077..zip`. You can see it here.

https://sourceforge.net/projects/wtl/files/WTL%2010/WTL%2010.0.10077

---

that makes me very uncomfortable lol